src/codemod.ts

```diff
@@ -0,0 +1,115 @@
+import { applyPatch } from 'diff'
+import * as sourcegraph from 'sourcegraph'
+import { Settings } from './settings'
+import { Change, Match, SourceTree } from './types'
+
+function sourcegraphURL(): URL {
+    const url =
+        sourcegraph.configuration.get<Settings>().value['pr.sourcegraphUrl'] ||
```

Comment:
I'm generally a little confused by a typescript thing for using `.get<Settings>().get('blah')` vs `.get<Settings>().value['blah']` vs `.get().value['blah']`. I cannot always use the first because I'll sometimes get an error like

```
Type 'string | Readonly<any>' is not assignable to type 'string'.
Type 'Readonly<any>' is not assignable to type 'string'.ts(2322)
```

Not sure why (but I haven't put in a lot of effort to figure out why).